conans/client/cmd/uploader.py

```diff
@@ -111,18 +111,32 @@ def _upload(self, conan_file, conan_ref, packages_ids, retry, retry_wait,
                                      "no packages can be uploaded")
             total = len(packages_ids)
             for index, package_id in enumerate(packages_ids):
-                ret_upload_package = self._upload_package(PackageReference(conan_ref, package_id),
+                pref = PackageReference(conan_ref, package_id)
+                cur_package_remote = self._registry.prefs.get(pref)
```

Comment:
Not sure about this, this is changing current behavior, and it will be again changed in conan 2.0. Probably remove this.

Reply:
Why is changing in Conan 2.0?